Sources/FluentDev/main.swift

```diff
@@ -0,0 +1,75 @@
+import Fluent
+
+print("Hello, Fluent!")
+
+//: Model (Active Record)
+
+User.first()
+User.last()
+
+User.take(50)
+User.find(1)
+User.find(1, 2, 4)
+User.findBy("name", .Equals, "Jane Doe")
+
+let u = User(deserialize: [:])
+u.save()
+u.delete()
+
+//: Query Builder - Different looks of using Query Builder Class
+
+// Insert
+
+Query<User>().insert(u.serialize()).run()
+
+// Update
+
+Query<User>().update(u.serialize()).run()
```

Comment:
When you call `.update()` or `.insert()` it should run it. 

Like `Query<User>().with("name", .Equals, "tanner").update("name", "new name")`